xarray/tests/test_dataarray.py

```diff
@@ -332,6 +332,10 @@ def test_constructor(self):
         expected = Dataset({None: (["x", "y"], data, {}, {"bar": 2})})[None]
         assert_identical(expected, actual)
 
+        actual = DataArray([1, 2, 3], coords={"x": [0, 1, 2]})
+        expected = DataArray([1, 2, 3], coords=[("x", [0, 1, 2])])
+        assert_identical(expected, actual)
```

Comment:
Can we also add a test for 2D dataarrays?

Reply:
Good point! I'll add one asap